scripts/content.js

```diff
@@ -35,8 +35,8 @@ fetch(proxyurl + url) // https://cors-anywhere.herokuapp.com/https://example.com
           linksList.forEach(function(product) {
             const node = product.childNodes[0];
             const label = node.getAttribute("aria-label");
-            let webbrand = label.split(" ")[0];
-            if(brand[0] === webbrand){
+            let webbrand = label.toLowerCase();
+            if(webbrand.includes(brand[0])){
                 product.style["border-color"] =resp.colourMap[resp.result[index].OverallScore];
                 product.classList.add('product');
```

Comment:
you can't get the right colour degree depending on the ```index``` here ```[resp.result[index].OverallScore]```, you have to get the degree from the original data for each brand by returning the whole data not only the brand name - regarding Matt's code above

Reply:
@shahenazmonia is this resolved?

Reply:
Yes :+1: